src/fragments/lib/graphqlapi/js/setup/connect.mdx

```diff
@@ -587,10 +550,11 @@ Join table records must be deleted prior to deleting the associated records. For
 #### Troubleshooting
 
 [comment]: # (EXPANDER SECTION to provide tips on how to avoid common errors such as: unauthorized errors and cross-platform gotchas. Each error should have a header with descriptive title to be included on a primary troubleshooting guide as well. Added Auth info from https://github.com/aws-amplify/docs/blob/main/src/fragments/lib/graphqlapi/js/mutate-data.mdx - not sure if this one should be for troubleshooting or general info added outside the expander.)
+[question]: #(RENE, is the following section part of Troubleshooting? The following section seems to be more focused on offroading than troubleshooting.) 
 
```

Comment:
Kept the Learn more eyebrow text instead of troubleshooting b/c this reads as more of an option. Is that correct or should we adjust the eyebrow text to say Troubleshooting in this instance? We originally had a note to provide troubleshooting for unauthorized errors and cross-platform gotchas - this does not read to me like it covers those errors. Additionally I think the intro for this expander needs to be updated to explain why it is where it is.